Gordon360/Documentation/Gordon360.xml

```diff
@@ -1,2785 +1,2734 @@
-            </summary>
-        </member>
-        <member name="M:Gordon360.Static.Methods.Helpers.GetCurrentSession(Gordon360.Models.CCT.Context.CCTContext)">
-            <summary>
-            Helper method that gets the current session we are in.
-            </summary>
-            <returns>The session code of the current session</returns>
-        </member>
-        <member name="M:Gordon360.Utilities.ImageUtils.DeleteImage(System.String)">
-            <summary>
-            Deletes an image from the filesystem, if there is one.
-            </summary>
-            <param name="imagePath">The path to which the image belonged.</param>
-        </member>
-        <member name="M:Gordon360.Utilities.ImageUtils.RetrieveImageFromPath(System.String)">
-            <summary>
-            Takes an image path and returns the data of the image at that path.
-            If given path is null it returns an empty string.
-            </summary>
-            <param name="imagePath">The path to the image</param>
-            <returns>The base64 data of the image</returns>
-        </member>
-        <member name="M:Gordon360.Utilities.ImageUtils.UploadImage(System.String,System.String,System.Drawing.Imaging.ImageFormat)">
-            <summary>
-            Uploads a news image
-            </summary>
-            <remarks>
-            Takes base64 image data and writes it to an image file. Note that if the target path
-            already has a file, the method will overwrite it (which gives no errors)
-            </remarks>
-            <param name="imagePath">The path to which the image belongs</param>
-            <param name="imageData">The base64 image data to be stored</param>
-            <param name="format">The format to save the image as. Defaults to Jpeg</param>
-        </member>
-        <member name="M:Gordon360.Utilities.ImageUtils.UploadImageAsync(System.String,Microsoft.AspNetCore.Http.IFormFile)">
-            <summary>
-            Uploads image from HTTP FormFile
-            </summary>
-            <remarks>
-            Takes image data and writes it to an image file. Note that if the target path
-            already has a file, the method will overwrite it (which gives no errors)
-            </remarks>
-            <param name="imagePath">The path to which the image belongs</param>
-            <param name="imageData">The image data to be stored</param>
-        </member>
-        <member name="M:Gordon360.Utilities.ImageUtils.GetBase64ImageDataFromPath(System.String)">
-            <summary>
-            Takes a filepath for an image, navigates to it, collects the raw data
-            of the file and converts it to base64 format. 
-            </summary>
-            <param name="imagePath">The path to the image</param>
-            <returns>The base64 content of the image</returns>
-        </member>
-    </members>
-</doc>
+<?xml version="1.0"?>

```

Comment:
I'm puzzled about why github is showing that every single line of this file has changed  Many seem the same, but some are different.   I can't find a commit that changes more than a small piece, and blame shows most lines haven't changed for weeks or months.  Do you know why it thinks every single line of this file changed?

Reply:
Yeah this also confuses me. I don't know why GitHub thinks every line of code in this file has been changed.. Some new code was added, such as mail location and user privacy, but most of the code has not been changed.